packages/nimble-components/src/rich-text/viewer/tests/rich-text-viewer.spec.ts

```diff
@@ -360,6 +345,106 @@ describe('RichTextViewer', () => {
                 'user:2'
             );
         });
+
+        it('should have valid states for valid configurations', () => {
```

Comment:
This test is a duplicate of "should have valid states by default". Remove it.

Reply:
Removed